src/test/pybind/test_rbd.py

```diff
@@ -762,6 +762,15 @@ def test_remove_snap(self):
         self.image.remove_snap('snap1')
         eq([], list(self.image.list_snaps()))
 
+    def test_remove_snap2(self):
+        eq([], list(self.image.list_snaps()))
```

Comment:
Nit: listing snaps is not needed (here and two lines below)

Reply:
thinks for your suggest

Reply:
@dillaman 
have updated